lib/Config.js

```diff
@@ -1320,6 +1321,18 @@ class Config extends EventEmitter {
         }
 
         this.lifecycleRoleName = config.lifecycleRoleName || null;
+        this.multiObjectDeleteConcurrency = constants.multiObjectDeleteConcurrency;
+        if (!isNaN(config.multiObjectDeleteConcurrency)) {
+            const extractedNumber = Number.parseInt(config.multiObjectDeleteConcurrency, 10);
+            if (extractedNumber > 0 && extractedNumber < 1000 && extractedNumber % 1 === 0) {
+                this.multiObjectDeleteConcurrency = extractedNumber;
+            }
+        }
+
+        this.multiObjectDeleteEnableOptimizations = true;
+        if (config.multiObjectDeleteEnableOptimizations === false) {
```

Comment:
```suggestion
        if (!config.multiObjectDeleteEnableOptimizations) {
```

Reply:
This was kept unchanged, because we want to enable the optimization by default, and this change would disable it, as there is no such configuration by default.